src/bosh-director/lib/bosh/blobstore_client/azurestoragecli_blobstore_client.rb

```diff
@@ -0,0 +1,138 @@
+require 'openssl'
+require 'digest/sha1'
+require 'base64'
+require 'securerandom'
+require 'open3'
+require 'json'
+
+module Bosh::Blobstore
+  class AzurestoragecliBlobstoreClient < BaseClient
+    # Blobstore client for azure storage account, using azure-storage-cli Go version
+    # @param [Hash] options azure storage account connection options
+    # @option options [Symbol] account_name
+    #   key that is applied before the object is sent to azure storage account
+    # @option options [Symbol, optional] account_key
+    # @option options [Symbol] azure_storage_cli_path
+    #   path to azure-storage-cli binary
+    # @option options [Symbol, optional] azure_storage_cli_config_path
+    #   path to store configuration files
```

Comment:
the `container_name` param docu is missing here.

Reply:
Done